migration/migration_blackbox_test.go

```diff
@@ -1404,10 +1406,29 @@ func testMigration110TrackerQueryID(t *testing.T) {
 func testMigration111WITinTrackerQuery(t *testing.T) {
 	migrateToVersion(t, sqlDB, migrations[:112], 111)
 	require.True(t, dialect.HasColumn("tracker_queries", "work_item_type_id"))
-
 	// check foreign key to work_item_types(id) exists
 	require.True(t, dialect.HasForeignKey("tracker_queries", "tracker_queries_work_item_type_id_fkey"))
+}
 
+func testMigration112CascadingDelete(t *testing.T) {
+	migrateToVersion(t, sqlDB, migrations[:113], 113)
+	checkForeignKeyConstraint := func(t *testing.T, table string, constraintName string) bool {
+		q := fmt.Sprintf("SELECT constraint_name FROM information_schema.table_constraints WHERE table_name = '%s' "+
```

Comment:
No, no, no :) There's something far more simple than this: 

```go
func testMigration112CascadingDelete(t *testing.T) {
	migrateToVersion(t, sqlDB, migrations[:113], 113)
	require.True(t, dialect.HasForeignKey("identities", "identities_user_id_fkey"))
	require.True(t, dialect.HasForeignKey("comment_revisions", "comment_revisions_modifier_id_fkey"))
	require.True(t, dialect.HasForeignKey("work_item_link_revisions", "work_item_link_revisions_modifier_id_fkey"))
	require.True(t, dialect.HasForeignKey("work_item_revisions", "work_item_revisions_modifier_id_fkey"))
}
```

And then you can get rid of all the rest where you're manually querying the information schema.

Reply:
fixed 05a94ce119818b9b2be4d5dbb84d332e45c3f194